### PR TITLE
added prettier config and prettified all codebase

### PR DESCRIPTION
**Pull request by alexandrtovmach**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -326,39 +313,35 @@ name and the semantic version is "1.2.3".
...
 ```
 ^(?P<major>0|[1-9]\d*)\.(?P<minor>0|[1-9]\d*)\.(?P<patch>0|[1-9]\d*)(?:-(?P<prerelease>(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+(?P<buildmetadata>[0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$
 ```
 
-And one with numbered capture groups instead (so cg1 = major, cg2 = minor,
-cg3 = patch, cg4 = prerelease and cg5 = buildmetadata) that is compatible
-with ECMA Script (JavaScript), PCRE (Perl Compatible Regular Expressions,
-i.e. Perl, PHP and R), Python and Go.
+And one with numbered capture groups instead (so cg1 = major, cg2 = minor, cg3 =
+patch, cg4 = prerelease and cg5 = buildmetadata) that is compatible with ECMA
```
Comment (jwdonahue) [inline index.md:326]: I like it better when the two operands are both on the same line.

**Comment by jwdonahue** [comment]
I like the idea of having some document formatting standards, particularly when they can be enforced by some sort of automation.  Can you please provide a link to the tooling used to format these documents?  Can it be configured with some sort of rule set?

Looking at some examples of the changes you have here, I don't see much value-add at this point.

**Comment by alexandrtovmach** [comment]
@jwdonahue Issue with more details #235 
>Can you please provide a link to the tooling used to format these documents?

https://prettier.io
> Can it be configured with some sort of rule set?
 
Yes, it can be configured by `.prettierrc` file

**Comment by alexandrtovmach** [comment]
> Looking at some examples of the changes you have here, I don't see much value-add at this point.

Probably yes, here is no any drastic changes (and I think it's okay), because this formatting needed only for a one standard

**Comment by alexandrtovmach** [comment]
@jwdonahue Would you approve this PR if there is no objections?

**Comment by jwdonahue** [comment]
Looking now, but my internet connection is very flaky the past few days, so it might be an few hours before I can reply...

**Comment by alexandrtovmach** [comment]
@jwdonahue No rush 😄 I also reviewed this changes, and decided to improve some thing, so let's see in a few days!

File: index.md (hunk 1 above)
Comment (jwdonahue) [reply index.md:326]: Is there anyway to fix this?

**Comment by jwdonahue** [comment]
Okay, I think except for the one case where an equals expression got split between lines, it looks awesome, but I am not a maintainer, so I can't actually approve it. 

I forgot my middle-school french about 2 minutes after I graduated, and except for a handful of programming languages, I am an English only speaker, so I can't evaluate how these changes rolled into the non-English files.  But, ya, if there were no objections from any of the relevant translators and it was up to me, I'd probably approve it.

And thank-you for adding tags!  I've been trying to encourage someone to go and tag everything for years now.

**Comment by alexandrtovmach** [comment]
Closed in favor of #266

**Pull request closed without merging**